### PR TITLE
feat: private keys support for testing

### PR DESCRIPTION
**Pull request by segfault-magnet**
closes: #160 

We'll still be using KMS in our e2e tests so that we remain as close to prod as possible.

The bridge repo can now specify their keys as "Private(key_here)" while we'll configure the keys as "Kms(arn_here)".

### CODE REVIEW

File: packages/eth/src/websocket.rs [hunk 1]
```diff
@@ -36,31 +75,123 @@ pub struct TxConfig {
...
+            Some(signer)
+        } else {
+            None
+        };
+
+        let l1_key = keys.main;
+        let main_signer = match l1_key {
+            L1Key::Kms(key) => Signer::make_aws_signer(&aws_client.expect("is set"), key).await?,
+            L1Key::Private(key) => Signer::make_private_key_signer(&key)?,
+        };
```
Comment (hal3e) [inline packages/eth/src/websocket.rs:168]: is a bit shorter
```suggestion
        let blob_signer = match keys.blob {
            Some(L1Key::Kms(key)) => {
                Some(Signer::make_aws_signer(aws_client.as_ref().expect("is set"), key).await?)
            }
            Some(L1Key::Private(key)) => Some(Signer::make_private_key_signer(&key)?),
            None => None,
        };

        let main_signer = match keys.main {
            L1Key::Kms(key) => Signer::make_aws_signer(&aws_client.expect("is set"), key).await?,
            L1Key::Private(key) => Signer::make_private_key_signer(&key)?,
        };
```

**Review by hal3e — APPROVED** [review]
Great work. Left some suggestions

File: packages/eth/src/websocket.rs [hunk 2]
```diff
@@ -24,10 +30,43 @@ pub struct WebsocketClient {
...
+    Private(String),
+}
+
+impl<'a> serde::Deserialize<'a> for L1Key {
+    fn deserialize<D>(deserializer: D) -> std::result::Result<Self, D::Error>
+    where
+        D: serde::Deserializer<'a>,
+    {
+        let value = String::deserialize(deserializer)?;
+        if let Some(k) = value.strip_prefix("Kms(").and_then(|s| s.strip_suffix(')')) {
```
Comment (MujkicA) [inline packages/eth/src/websocket.rs:45]: A brief comment on how to setup the key env somewhere would be nice.
My preference would've been to keep separate env vars so their way of use is more obvious, but not worth it at this point

**Review by MujkicA — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -40,3 +40,191 @@ Before running the script, ensure you have the following installed:
 
 - [Docker](https://www.docker.com/get-started) (required for running SchemaSpy)
 - [Sqlx CLI](https://github.com/launchbadge/sqlx/tree/main/sqlx-cli) (required for running migrations)
+
+## Configuration
```
Comment (MujkicA) [inline README.md:44]: went above and beyond 😁

**Review by MujkicA — APPROVED** [review]

**Pull request merged**